CHANGELOG.md

```diff
@@ -2,9 +2,12 @@
 
 ## Unreleased
 
+- SC-2661 Add Light version of button primary
+
 ## 1.0.26
 
 - SC-2527 Add DarkHelpTooltip component and allow to customize inner question mark color of HelpIcon
+- SC-2527 Add "white" and "transparentBlack" to the theme
```

Comment:
Those 2 variables were added in the previous version but not documented in the changelog, it's better if they appear in it since you should know that you have to add theme in SQ theme once you update past this version.